simvue/run.py

```diff
@@ -1005,96 +1059,48 @@ def save(
             self._error("Cannot upload output files for runs in the created state")
             return False
 
-        is_file: bool = False
-
-        if isinstance(filename, str):
-            if not os.path.isfile(filename):
-                self._error(f"File {filename} does not exist")
```

Comment:
You removed this, but think it will still be useful to check if the file exists and raise an error if it does not

Reply:
`pydantic.FilePath` requires the file to exist